src/arb/acb_mat.jl

```diff
@@ -747,8 +746,7 @@ end
 ###############################################################################
 
 function (x::AcbMatSpace)()
```

Comment:
Currently we already have a default method for this which is
```
function (a::MatSpace{T})() where {T <: NCRingElement}
   return zero_matrix(base_ring(a), nrows(a), ncols(a))::dense_matrix_type(T)
end
```
but that means we implicitly expect `zero_matrix` to be implemented... With the new system both can be implemented generically.